### PR TITLE
chore: Bring back babel-plugin-react-intl

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: src/i18n/en-US.json [hunk 1]
```diff
@@ -922,7 +922,7 @@
   "ssoLogin.pasteButton": "Paste code",
   "ssoLogin.subhead": "Enter the company SSO access code.",
   "ssoLogin.subheadCode": "Please enter your SSO code",
-  "ssoLogin.subheadCodeOrEmail": "Please enter your email or SSO code",
+  "ssoLogin.subheadCodeOrEmail": "Please enter your email or SSO code.",
```
Comment (ffflorian) [inline src/i18n/en-US.json:925]: Not part of the PR I think :slightly_smiling_face:

Comment (Yserz) [reply src/i18n/en-US.json:925]: It is part of this PR because this is a change that wasn't propagated but should have.

Comment (ffflorian) [reply src/i18n/en-US.json:925]: Ah okay

**Review by ffflorian — APPROVED** [review]
